ivy/functional/frontends/jax/numpy/logic.py

```diff
@@ -285,5 +285,30 @@ def setxor1d(ar1, ar2, assume_unique=False):
     return ret
 
 
+@to_ivy_arrays_and_back
+def unpackbits(x, /, *, axis=None, count=None, bitorder="big"):
+    bits = ivy.arange(8, dtype="uint8")
+    if bitorder == "big":
+        bits = bits[::-1]
+    if axis is None:
+        x = ivy.flatten(x)
+        axis = 0
+    x = ivy.swapaxes(x, axis, -1)
+
+    unpacked = (
+        (x[..., None] & ivy.expand_dims(bits, axis=tuple(range(x.rdim)))) > 0
```

Comment:
It seems that `x.rdim` is causing all the test failures so far. Could you please fix it?